test/rate_by_service_sampler_test.rb

```diff
@@ -1,5 +1,5 @@
 require 'minitest'
-require 'ddtrace/sampler'
+require 'ddtrace'
```

Comment:
Turns out `test/default_test.rb`'s `require 'ddtrace'` was **THE** require statement that other minitest tests in the `test:main` suite depended on, as a leaky side-effect.

The only affected test is `test/rate_by_service_sampler_test.rb`, so I properly required the whole tracer in it, as is required.